### DISCUSSION TITLE
How to use Bing-AI as model in the code ?

### DISCUSSION THREAD

**Discussion by RajeshTechForge** [xtekky/gpt4free#1675] [Q&A] [answered]
```
client = Client()
response = client.chat.completions.create(
    model=" ",
    messages=[{"role": "user", "content": "hello"}])

print(response.choices[0].message.content)
```
How to use Bing AI as model in the code ?

**Comment by hlohaus** [comment, marked as answer]
The model can you left blank.
In the client can you define the a provider for your requests.

Example:

client = Client(provider=g4f.Provider.Bing)

**Reply by RajeshTechForge** [reply]
Thanks

**Comment by julianponguta** [comment]
If I used the OpenAI library, is it possible to do this?

**Reply by RajeshTechForge** [reply]
**Yes, you can achieve this** with the OpenAI library, but the code will be different. Refer to the OpenAI library documentation.

If you want to use your OPENAI API Key, I highly recommend using **LangChain**.
